awx_collection/tests/integration/targets/tower_project_manual/tasks/create_project_dir.yml

```diff
@@ -0,0 +1,57 @@
+- name: get tower host variable
+  shell: tower-cli config host | cut -d ' ' -f2
```

Comment:
This is gross but im not sure how to do it otherwise, and we need tower-cli for the stuff that follows anyways

Reply:
`| awk '{print $2}'` 🤷‍♂